menu/drivers/rgui.c

```diff
@@ -1616,6 +1616,7 @@ static bool rgui_fonts_init(rgui_t *rgui)
       case RETRO_LANGUAGE_FINNISH:
       case RETRO_LANGUAGE_INDONESIAN:
       case RETRO_LANGUAGE_SWEDISH:
+   /* case RETRO_LANGUAGE_CZECH: */
```

Comment:
why is it commented?

Reply:
> Unfortunately the "extended ASCII" font for RGUI does not cover all Czech characters, thus RGUI is currently incompatible with this language.

I left it there as a sort of reminder that one might want to update/extend the font. Should I remove it?